__init__.py

```diff
@@ -478,20 +480,22 @@ def exec_exists(bin):
     except:
         return False
 
+
 def compute_size(maxs, imgw, imgh):
     if imgw > imgh:
         return maxs, maxs*imgh/imgw
     else:
-        return maxs*imgw/imgh, maxs
+        return maxs *imgw/imgh, maxs
+
 
 if __name__ == "__main__":
     from kivy.base import runTouchApp
     from kivy.uix.boxlayout import BoxLayout
     from kivy.uix.label import Label
-
+    
     box = BoxLayout(orientation="vertical")
     fileChooser = FileChooserThumbView(thumbsize=128)
-    label = Label(markup=True, size_hint_y=None)
+    label = Label(markup=True, size_hint=(1, 0.05))
```

Comment:
Can you explain why this is needed?

Reply:
The height of the label widget is to0 big , thus taking unnecessary space for display of selected path, and also the files or directories displayed are few since half of the height is taken by label widget